libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -414,6 +422,54 @@ export class WorkspaceInstallManager extends BaseInstallManager {
 
     // TODO: Remove when "rush link" and "rush unlink" are deprecated
     LastLinkFlagFactory.getCommonTempFlag(this.rushConfiguration).create();
+
+    // Stage 2 rebuild pending
+    if (this.twoStageInstall && !this.options.ignoreScripts) {
+      const rebuildArgs: string[] = ['rebuild', '--pending', '--recursive'];
+
+      if (
+        this.rushConfiguration.pnpmOptions.pnpmStore === 'local' ||
+        EnvironmentConfiguration.pnpmStorePathOverride
+      ) {
+        // --store is not supported to pnpm rebuild... So, config.store is used here
+        rebuildArgs.push(`--config.storeDir=${this.rushConfiguration.pnpmOptions.pnpmStorePath}`);
+      }
+
+      if (this.options.collectLogFile) {
+        rebuildArgs.push('--reporter', 'ndjson');
```

Comment:
Will the logfile overwrite the existing log? Or is this to output logs into the console, which we subsequently append into the log we write?

Reply:
later one. :)